plugins/binance/binance_cli.go

```diff
@@ -0,0 +1,25 @@
+package binance
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func BinanceCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Binance CLI", 
+		Runs:      []string{"binance-cli"},
+		DocsURL:   sdk.URL("https://binance.com/docs/cli"), 
```

Comment:
This link does not seem to be a valid one. 

<img width="1258" alt="image" src="https://github.com/1Password/shell-plugins/assets/18581859/b0f2b240-1b9c-4ec1-926f-177f4d89943a">

Reply:
Thanks for pointing out this, updated with correct link